R/modules.R

```diff
@@ -334,17 +447,11 @@ print.teal_module <- function(x, ...) {
   invisible(x)
 }
 
-
 #' @rdname teal_modules
 #' @export
```

Comment:
@vedhav maybe add some examples for print/format? Take the one you proposed in the opening of this PR